examples/tutorials/pruning_bert_glue.py

```diff
@@ -159,11 +159,21 @@ def preprocess_function(examples):
 from datasets import load_metric
 from transformers.modeling_outputs import SequenceClassifierOutput
 
+from nni.common.version import torch_version_is_2
+
+
+if torch_version_is_2():
+    from torch.optim.lr_scheduler import LRScheduler
+    SCHEDULER = LRScheduler
+else:
+    from torch.optim.lr_scheduler import _LRScheduler
+    SCHEDULER = _LRScheduler
```

Comment:
what about put this `if ... else ...` in some where and we just `from xxx import SCHEDULER` here, then we don't need to copy paste it everywhere.